ietfYangDraftPull/draftPullUtility.py

```diff
@@ -108,52 +108,66 @@ def check_early_revisions(directory: str, LOGGER: logging.Logger) -> None:
         :param directory    (str) full path to directory with yang modules
         :param LOGGER       (logging.Logger) formated logger with the specified name
     """
-    for f in os.listdir(directory):
-        # Extract the YANG module name from the filename
-        module_name = f.split('.yang')[0].split('@')[0]   # Beware of some invalid file names such as '@2015-03-09.yang'
+    get_module_name = lambda filename: filename.split('.yang')[0].split('@')[0]
```

Comment:
A full function definition would be more easily visible than assigning lambdas to variables. Also makes it easier to add type hints.

Reply:
replaced lambda with ordinary function

Reply:
same for `is_revision_part_valid`

Reply:
fixed